bigframes/display/anywidget.py

```diff
@@ -23,13 +23,14 @@
 import pandas as pd
 
 import bigframes
+from bigframes.core.blocks import PandasBatches
```

Comment:
> Use import statements for packages and modules only, not for individual types, classes, or functions.

https://google.github.io/styleguide/pyguide.html#22-imports

```suggestion
from bigframes.core import blocks
```